nix/default.nix

```
@@ -70,7 +70,7 @@ stdenv.mkDerivation rec {
   postInstall = ''
     # xorg.xrandr needed for LWJGL [2.9.2, 3) https://github.com/LWJGL/lwjgl/issues/128
     wrapQtApp $out/bin/polymc \
-      --set GAME_LIBRARY_PATH ${gameLibraryPath} \
+      --set LD_LIBRARY_PATH ${gameLibraryPath} \
```

I believe this should be
```suggestion
      --prefix LD_LIBRARY_PATH : ${gameLibraryPath} \
```
instead, since then it's more hackable (a user troubleshooting a mod that's missing a dependency can run `LD_LIBRARY_PATH=/nix/store/.../lib polymc` to get it to work.)